### PR TITLE
Add sleek v1.0.8

### PR DESCRIPTION
**Pull request by ransome1**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by ransome1** [comment]
Dear reviewers.

I (the maintainer and one of the developers of sleek) understand this project has not reached 30 watchers yet, but it has already thrice the stars that are the minimum requirement and almost 30 forks. sleek is being actively developed and I'm sure quite some users would appreciate the app to be available on brew.

File: Casks/sleek.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  url "https://github.com/ransome1/sleek/releases/download/v#{version}/sleek-#{version}-mac.dmg"
+  name "sleek"
+  desc "Todo app based on todo.txt, free and open-source"
+  homepage "https://github.com/ransome1/sleek"
+
+  livecheck do
+    url "https://github.com/ransome1/sleek/releases.atom"
+    strategy :git
+  end
```
Comment (bevanjkay) [inline Casks/sleek.rb:13]: @ransome1 The livecheck stanza can be removed as it is implied when the URL stanza is from Github.

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/sleek.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  app "sleek.app"
+
+  zap trash: "~/Library/Application Support/sleek"
```
Comment (bevanjkay) [inline Casks/sleek.rb:17]: There are some files/folders missing from the zap stanza

```suggestion
zap trash: [
    "~/Library/Application Support/sleek",
    "~/Library/Saved Application State/RobinAhle.sleektodomanager.savedState",
    "~/Library/Preferences/RobinAhle.sleektodomanager.plist",
  ]
```

**Review by bevanjkay — DISMISSED** [review]

File: Casks/sleek.rb (hunk 2 above)
Comment (ransome1) [reply Casks/sleek.rb]: Thanks!

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thanks @ransome1

**Comment by ransome1** [comment]
@bevanjkay thanks for your help!

**Pull request merged**